src/librustdoc/html/static/js/main.js

```diff
@@ -377,7 +378,10 @@ function loadCss(cssUrl) {
         }
         ev.preventDefault();
         searchState.defocus();
-        window.hidePopoverMenus();
+        // If the notable traits popover is open, and the user presses Escape,
+        // reset focus back to the link.
+        hideNotable(true);
```

Comment:
Since it's already called inside `hideAllModals`, why not this call (with the `true` argument) in there directly? Since we know if the opened modal is a notable one or not, we can put a check for that directly in there too.

What do you think?

Reply:
I guess it would make more sense rather than calling `hideNotable` multiple times.